packages/playground/src/app.tsx

```diff
@@ -194,14 +196,25 @@ export const OutputView: FunctionComponent<OutputViewProps> = (props) => {
       } else {
         setViewSelection({ type: "file", filename: "", content: "" });
       }
+    } else if (viewSelection.type ===  "swagger-ui" ){
+        if (props.outputFiles.length > 0) {
+          void loadSwaggerUIforFile(props.outputFiles[0]);
```

Comment:
Not sure what `void` is doing here?

Reply:
Maybe to suppress the eslint warning about a promise being left "floating"? We have the same pattern above for loadOutputFile.

Reply:
Blah. I'm sad about satisfying eslint with actual syntax that A) doesn't change the semantics of the program in any observable way, and B) uses an operator whose primary usecase is minifiers trying to shave 3 bytes off of a references to `undefined`. But I won't block since a comment would also be annoying, and I'm assuming we don't want to actually `await` this for reasons.